ecomap/src/python/ecomap/config.py

```diff
@@ -0,0 +1,52 @@
+from ConfigParser import SafeConfigParser
+from time import time
+import logging
+
+password = ['password', 'pass', 'pswd']          # keys which must be string
```

Comment:
Why do you need this line? Could you please explain? What do you think happen if user will set password as argument 'db_pass' in ecomap.conf?